src/multivariate/product.jl

```diff
@@ -41,6 +41,27 @@ cov(d::Product) = Diagonal(var(d))
 entropy(d::Product) = sum(entropy, d.v)
 insupport(d::Product, x::AbstractVector) = all(insupport.(d.v, x))
 
+"""
+    minimum(d::Product)
+
+Return the minimum of the support of each dimension of `d`.
+"""
+minimum(d::Product) = minimum.(d.v)
```

Comment:
Maybe use `map` instead of broadcasting? It seems broadcasting is not needed here, and so it might be good to avoid the heavy broadcasting machinery.

Reply:
```suggestion
minimum(d::Product) = map(minimum, d.v)
```

Reply:
Doesn't need a docstring then, just an implementation